app.py

```diff
@@ -129,9 +147,55 @@ def reply():
                 name=paymentThreadName, target=paymentUtils.paymentStatusCheck, args=(paymentID, sender))
             paymentStatusThread.start()
             return str(reply_text)
-    
 
+        elif session.get('lastState') == 'nu.enterName':
+            session['userFirstName'] = incoming_msg
+            response = "Hey {} ! What's your last name ?".format(session.get('userFirstName'))
+            session['lastState'] = 'nu.enterLastName'
+            reply_text = MessagingResponse()
+            reply_text.message(response)
+            return str(reply_text)
+        
+        elif session.get('lastState') == 'nu.enterLastName':
+            session['userLastName'] = incoming_msg
+            response = "Hey {} {} ! What's your email id ?".format(session.get('userFirstName'), session.get('userLastName'))
+            session['lastState'] = 'nu.enterEmail'
+            reply_text = MessagingResponse()
+            reply_text.message(response)
+            return str(reply_text)
+        
+        elif session.get('lastState') == 'nu.enterEmail':
+            session['userEmail'] = incoming_msg
+            response = "Could you please tell us under what category do you fall under ? \n \n1. Manufacturer \n2. Wholeseller or Distributor \n3. Retailer \n4. Electrical Contractor \n5. Customer Please Enter the Category Number"
+            session['lastState'] = 'nu.enterCategory'
+            session['lastMenu'] = 'nu.enterCategoryMenu'
+            reply_text = MessagingResponse()
+            reply_text.message(response)
+            return str(reply_text)
            
+        elif session.get('lastState') == 'nu.enterShippingAddress':
+            session['userShippingAddress'] = incoming_msg
+            response = "Could you please enter the default Billing Address ?"
```

Comment:
Here we need to ask 'Do you want to keep your billing address same as your shipping address'?
Options : 
Yes -> Go to the next step
No -> 'Please enter your billing address' -> Enter the address -> Next step